product/swagger/product.yaml

```diff
@@ -296,4 +296,63 @@ paths:
           description: Produto não encontrado
           content:
             application/json:
-              example: Este produto não foi encontrado
\ No newline at end of file
+              example: Este produto não foi encontrado
+  /api/products/listByPrice:
+    parameters:
+    - name: max
+      in: query
+      schema:
+        type: number
+      required: true
+    - name: min
+      in: query
+      schema:
+        type: number
+      required: true   
+    get:
+      summary: Lista os produtos por faixa de preço
+      responses:
+        200:
+          description: Sucesso
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/products"
```

Comment:
i think you need to review your references

like on #11

Reply:
done